infra/app/web.bicep

```diff
@@ -0,0 +1,18 @@
+param environmentName string
+param location string = resourceGroup().location
+param appServicePlanId string
+
+param serviceName string = 'web'
+
+module web '../core/host/appservice-dotnet.bicep' = {
+  name: '${serviceName}-appservice-dotnet-module'
```

Comment:
Also, did you have to make any changes to bicep/core for this?

Reply:
We created templates(`sqlserver-catalog.bicep` and `sqlserver-identity.bicep`) in sqlserver for `catalog` and `identity` separately, we didn't need to make any changes to bicep/core other than that.

Reply:
I removed these: https://github.com/zedy-wj/eShopOnWeb/pull/2